src/test/Testcafe/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "UI tests for Altinn tjenester 3.0",
   "main": "app.js",
   "scripts": {
-    "test": "node .\\testRunner --env=dev"
+    "test": "node .\\testRunner --env=dev",
```

Comment:
Will this include the wcag tests also ? if yes, should it be modified such that wcag tests are not run here ?

Reply:
The new pipeline will trigger wcag tests by running ```npm run wcag``` not by running the ```npm test``` script.